google-cloud-storage/src/main/java/com/google/cloud/storage/StorageException.java

```diff
@@ -73,4 +82,25 @@ public static StorageException translateAndThrow(RetryHelperException ex) {
     BaseServiceException.translate(ex);
     throw new StorageException(UNKNOWN_CODE, ex.getMessage(), ex.getCause());
   }
+
+  /**
+   * Translate IOException to the StorageException that caused the error. This method defaults to
+   * idempotent always being {@code true}. This method retries the transient issues Connection
+   * Closed Prematurely and Connection Reset.
+   *
+   * <p>Review {@code RETRYABLE_ERRORS} for a full list of retryable errors.
+   *
+   * @returns {@code StorageException}
+   */
+  public static StorageException translate(IOException exception) {
```

Comment:
`translate` is an uncommon name for this operation. Perhaps `wrap`?

Reply:
Translate is what's being used in the code right now and wanted to keep consistent given there's already `translateAndThrow`